src/core/zowe/core_for_zowe_sdk/validators.py

```diff
@@ -12,9 +12,12 @@
 
 import commentjson
 from jsonschema import validate
+import os
+from typing import Union
+import requests
 
 
-def validate_config_json(path_config_json: str, path_schema_json: str):
+def validate_config_json(path_config_json: Union[str, dict], path_schema_json: str):
```

Comment:
Is it ok to make the paht_schema_json: str optional?
And maybe we just pull it from the config file?